### PR TITLE
Fixed styles for hidden layers

### LINKED ISSUE #9771: Correct style for some layer elements when its hidden
Hi guys! 

This is an enhancement of the style of a layer when it's hidden so it's more clear that it is.  

The layer title should be in #CBCED0 and the icon in 24% transparency. 

artboard

Let me know if you have any question :) 
and thanks!

### PR DESCRIPTION
**Pull request by MariaCheca**
Fixes #9771 

screen shot 2016-10-03 at 17 05 09

CR @javierarce please :)

### CODE REVIEW

**Comment by javierarce** [comment]
The spec `editor/layers/data-layer-view should be displayed as hidden when layer is hidden` failed, right?

File: app/assets/stylesheets/editor-3/inline-editor.css.scss [hunk 1]
```diff
@@ -19,3 +19,6 @@
   @include flex(1);
   width: 0;
 }
+.Inline-editor-text.is-hidden {
+  opacity: 0.24;
```
Comment (piensaenpixel) [inline app/assets/stylesheets/editor-3/inline-editor.css.scss]: @include opacity()

File: app/assets/stylesheets/editor-3/layers-list.css.scss [hunk 2]
```diff
@@ -123,6 +123,9 @@
   border-radius: $baseSize / 2;
   cursor: pointer;
 }
+.Editor-ListLayer-media.is-hidden {
+  opacity: 0.24;
+}
```
Comment (piensaenpixel) [inline app/assets/stylesheets/editor-3/layers-list.css.scss]: same here

**Comment by MariaCheca** [comment]
@javierarce yes, currently I'm fixing a bunch of tests in a couple of branches, I'll fix these ASAP

File: app/assets/stylesheets/editor-3/inline-editor.css.scss (hunk 1 above)
Comment (MariaCheca) [inline app/assets/stylesheets/editor-3/inline-editor.css.scss]: Is it still necessary? Take a look at this table, looks like it's supported by all browsers now: http://caniuse.com/#feat=css-opacity

Comment (piensaenpixel) [inline app/assets/stylesheets/editor-3/inline-editor.css.scss]: ok, the i'll prepare a branch to delete the mixin

Comment (MariaCheca) [inline app/assets/stylesheets/editor-3/inline-editor.css.scss]: Nice! thank you :)

**Comment by javierarce** [comment]
:+1:

**Pull request merged**